tests.py

```diff
@@ -8,12 +8,28 @@
 import os
 import unittest
 from subprocess import Popen, PIPE
+from ipc.loader import load_plugin
 
 
 RULES_PATH = os.path.abspath(os.path.join('.', 'sample_rules'))
 
+def send(signal, value):
+    "Function is overriden by plugin"
+
+def receive():
+    "Function is overriden by plugin"
+
+def format_ipc_input(data):
+    return [ tuple(elm.split(' = ')) for elm in data.split('\n') ]
+
+def format_ipc_output(data):
+    return (('', '') if data == '' else tuple(data.strip().split(' = ')))
```

Comment:
Because the expected output string is canonical (the 'official' standard), and it is easier to see differences in strings. In the future this will change a bit, since we will no longer use the signal emissions as 'log' and then you may have a check for the IPC emit and one for the string in the log. For now we're considering both the same, and the string is canonical.